sci-electronics/drahnr-oregano/drahnr-oregano-0.84.40.ebuild

```diff
@@ -17,29 +18,30 @@ KEYWORDS="amd64 ppc x86"
 
 S="${WORKDIR}/oregano-${PV}"
 
-COMMON_DEP="dev-libs/glib:2
+COMMON_DEPEND="
+	dev-libs/glib:2
 	dev-libs/libxml2:2
 	x11-libs/goocanvas:2.0
 	x11-libs/gtk+:3
 	x11-libs/gtksourceview:3.0"
 
-DEPEND="${COMMON_DEP}
-	${PYTHON_DEPS}
+DEPEND="${COMMON_DEPEND}
+	${PYTHON_DEPS}"
```

Comment:
`${PYTHON_DEPS}` in `DEPEND` for `python-any-r1`? :stuck_out_tongue:

Reply:
Done :tada: